vignettes/finalsize.Rmd

```diff
@@ -46,23 +46,14 @@ knitr::opts_chunk$set(
 )
 ```
 
-## Getting _finalsize_
-
-The _finalsize_ package is currently available on [Github](https://github.com/epiverse-trace/finalsize), and can be installed using the `remotes` or `devtools` packages.
-
-```r
-# first install the remotes packages
-install.packages("remotes")
-
-# install finalsize using remotes
-remotes::install_github("epiverse-trace/finalsize")
-```
-
 ```{r}
+# load finalsize
 library(finalsize)
 
 # load necessary packages
-library(data.table)
+if (!require("socialmixr")) install.packages("socialmixr")
+if (!require("ggplot2")) install.packages("ggplot2")
+
```

Comment:
This only needs one more line for
`if(!require("data.table")) install.packages("data.table")`

Reply:
Oh, I just check that it is not required because of `base::merge()` here https://github.com/epiverse-trace/finalsize/pull/121/files#diff-00c8410963081d3f4414fefb13ef6ded91465dfb60bf79d3568cd7fc47542442R237